src/styles/cylc/_drawer.scss

```diff
@@ -40,3 +51,7 @@
     padding-right: 15px;
   }
 }
+
+.v-navigation-drawer--close {
+  visibility: unset !important;
```

Comment:
The main panel has higher z-index, so it's displayed if the drawer is still visible.